Detectors/FIT/base/include/FITBase/Geometry.h

```diff
@@ -0,0 +1,88 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef ALICEO2_FIT_GEOMETRY_H_
+#define ALICEO2_FIT_GEOMETRY_H_
+////////////////////////////////////////////////
+// Full geomrtry  hits classes for detector: FIT    //
+////////////////////////////////////////////////
+
+#include <Rtypes.h>
+#include <TNamed.h>
+namespace o2
+{
+namespace FIT
+{
+
+class Geometry : public TNamed
```

Comment:
Is the class using any functionality of `TNamed` or could it be independent?

Reply:
Please drop this inheritance.